easybuild/easyconfigs/p/Python/unixccompiler-Python-2.7.11.patch

```diff
@@ -0,0 +1,11 @@
+--- EasyBuildTesting/Python-2.7.9/Lib/distutils.orig/unixccompiler.py	2014-12-10 16:59:34.000000000 +0100
```

Comment:
I've already added this to the Python .eb file, so do you want comments on both patch and eb file?
```
# patch needed for building h5py
patches = ['unixccompiler-Python-2.7.11.patch']
```